cpgvalidator/src/main/scala/io/shiftleft/cpgvalidator/facts/InFactsImporter.scala

```diff
@@ -23,28 +23,28 @@ class InFactsImporter extends FactsImporter {
       NodeTypes.LITERAL has 1 incoming EdgeTypes.AST from SuperTypes.Expression or NodeTypes.CONTROL_STRUCTURE,
       NodeTypes.LITERAL has 0 to N incoming EdgeTypes.CFG from SuperTypes.Expression or NodeTypes.METHOD,
       NodeTypes.LITERAL has 0 to 1 incoming EdgeTypes.RECEIVER from NodeTypes.CALL,
-      NodeTypes.LITERAL has 0 to 1 incoming EdgeTypes.ARGUMENT from NodeTypes.CALL or NodeTypes.RETURN,
+      NodeTypes.LITERAL has 0 to 1 incoming EdgeTypes.ARGUMENT from NodeTypes.CALL or NodeTypes.RETURN or NodeTypes.UNKNOWN,
```

Comment:
Why do we need to put UNKNOWN here if we special case it in the checker?

Reply:
I removed that special filtering in the checker. Otherwise, UNKNOWN nodes would not  be taken into account when checking edges.

Reply:
You are right the filtering at this point is wrong. But please have a look at the `validateAllDstNodeTypes` method. There we want to also allow UNKNOWN nodes.

Reply:
True! Fixed that with the latest commits here.
With that, no changes to the facts are needed.